test-data/unit/check-enum.test

```diff
@@ -619,8 +619,8 @@ reveal_type(A.x.name)   # N: Revealed type is "Literal['x']?"
 reveal_type(B.a.name)   # N: Revealed type is "Literal['a']?"
 
 # TODO: The revealed type should be 'int' here
```

Comment:
You could remove this I guess

Reply:
Oh, I didn't notice there are several copies of that TODO comment, thanks!